src/batch/src/executor/insert.rs

```diff
@@ -119,6 +126,13 @@ impl InsertExecutor {
                 columns = ordered_cols
```

Comment:
I think there might be a segmentation fault at line: 124
if the NULL padding is still not executed here, then the `columns` might not have enough capacity to hold the values at the correct index

Reply:
Thanks for pointing this out! I believe it's resolved now :)